pytest.ini

```diff
@@ -1,5 +1,3 @@
 [pytest]
 norecursedirs=dist build .tox .eggs
-addopts=--doctest-modules --flake8 --black --cov
-doctest_optionflags=ALLOW_UNICODE ELLIPSIS
```

Comment:
Reckless abandon.  Uses Windows.  Likes all our pretty songs, but knows not what it means.

Reply:
BTW, JP merges my fork of [jaraco/skeleton](https://blog.jaraco.com/a-project-skeleton-for-python-projects/) into this repo for a lot of these boilerplate files.  This is one of the skeleton files that we don't touch here -- we change them in the skeleton and then merge here to pickup changes to the skeleton.